include/GafferScene/RenderController.h

```diff
@@ -96,7 +96,26 @@ class GAFFERSCENE_API RenderController : public boost::signals::trackable
 			SetsGlobalComponent = 2,
 			RenderSetsGlobalComponent = 4,
 			CameraOptionsGlobalComponent = 8,
-			AllGlobalComponents = GlobalsGlobalComponent | SetsGlobalComponent | RenderSetsGlobalComponent | CameraOptionsGlobalComponent
+			TransformBlurGlobalComponent = 16,
+			DeformationBlurGlobalComponent = 32,
+			AllGlobalComponents = GlobalsGlobalComponent | SetsGlobalComponent | RenderSetsGlobalComponent | CameraOptionsGlobalComponent | TransformBlurGlobalComponent | DeformationBlurGlobalComponent
+		};
+
+		struct MotionBlurOptions
+		{
+			MotionBlurOptions() : transformBlur( false ), deformationBlur( false ), shutter( 0 )
+			{
+			}
+
+			MotionBlurOptions( const IECore::CompoundObject *globals, const ScenePlug *scene );
+
+			bool operator==( const MotionBlurOptions &other )
+			{
+				return transformBlur == other.transformBlur && deformationBlur == other.deformationBlur && shutter == other.shutter;
+			}
+			bool transformBlur;
+			bool deformationBlur;
+			Imath::V2f shutter;
```

Comment:
I wonder if this is a place where the inline initialisers for data members are worth using? Maybe makes the definition a little more concise?

```
struct MotionBlurOptions
{
	MotionBlurOptions() = default;
	MotionBlurOptions( const IECore::CompoundObject *globals, const ScenePlug *scene );
	bool transformBlur = false;
	bool deformationBlur = false;
	Imath::V2f shutter = Imath::V2f( 0 );
};
```

Reply:
Done in 6d00f94